src/renderer/components/data-settings/data-settings.js

```diff
@@ -1303,7 +1303,7 @@ export default Vue.extend({
 
     getChannelInfoLocal: function (channelId) {
       return new Promise((resolve, reject) => {
-        ytch.getChannelInfo(channelId, 'latest').then(async (response) => {
```

Comment:
not sure why 'latest' was provided here?

Reply:
Looks like a leftover from an early revision. Doesn't seem like it's needed.